test_runner/src/main/kotlin/ftl/args/ValidateCommonArgs.kt

```diff
@@ -8,13 +8,22 @@ fun CommonArgs.validate() {
     assertShardTime()
     assertRepeatTests()
     assertSmartFlankGcsPath()
+    assertOrientationCorrectness()
+}
+
+private fun CommonArgs.assertOrientationCorrectness() {
+    val orientationsList = listOf("portrait", "landscape", "default")
```

Comment:
This is the crux of the logic. The question here is hardcoded values ok? Or shall it actually go ahead and try to retrieve the values that are available? 
WDTTT?

Reply:
On the one hand, I think is ok because orientation values shouldn't be changed, but if you want to fetch this parameter from API look on AndroidCatalog.IosCatalog.supportedOrientationsAsTable

Reply:
@adamfilipow92 it what i was thinking too. In this case it might be fine as orientations for phones should not change.